config.json

```diff
@@ -1172,7 +1631,19 @@
         "slug": "rail-fence-cipher",
         "name": "Rail Fence Cipher",
         "uuid": "3daeb229-0638-443f-9ae2-222ebb88c11a",
-        "prerequisites": [],
+        "prerequisites": [
+          "strings",
+          "cond",
+          "case",
+          "if",
+          "multiple-clause-functions",
+          "guards",
+          "pattern-matching",
+          "recursion",
+          "enum",
+          "ranges",
+          "streams"
```

Comment:
A lot of solutions use `Stream.cycle`. But the example one doesn't... I'm not sure about this one.